twarep/electrifly
# Migrated SOC Graph Data Input from Test Files to Server Database

## Checklist

## What I Did
- I edited two files: `app.py`, and `flight_querying`.
- I deleted the `test_data/` folder because of the migration.
- I added a new file called `Graphing.py`, which will be where all the graphing code will be stored.

## How I Did it?
- I used the `matplotlib Python` library to graph the data. I used the `SQLAlchemy create_engine` function to connect to the server database.

## How to Test?
- Pull the newest changes from this branch. Ensure you are on university wifi, or are connected through VPN.
- Run the `app.py` script and check that the newest data in the SOC graph is showing (August flights).
- Then, check if you can select one/multiple flights and see how the graph changes with your selection (the legend too).
